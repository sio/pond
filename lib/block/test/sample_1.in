# Block based template engine

## Sample usage

<!--SECTION Some special characters (​ 世界) START-->
<!--SECTION Some special characters (​ 世界) END-->

Text outside of section blocks will be rendered as is.

<!--SECTION SimpleField START-->
Text inside section blocks will be overwritten.
<!--SECTION SimpleField END-->
