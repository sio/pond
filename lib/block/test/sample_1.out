# Block based template engine

## Sample usage

<!--SECTION Some special characters (​ 世界) START-->
Special
Characters
Rendered
Correctly:
​ 世界
<!--SECTION Some special characters (​ 世界) END-->

Text outside of section blocks will be rendered as is.

<!--SECTION SimpleField START-->
hello
world
<!--SECTION SimpleField END-->
